### PR TITLE
Update Metalsmith and move to Dart Sass

### PR DESCRIPTION
**Pull request by vanitabarrett**
Came up as part of https://github.com/alphagov/govuk-design-system/issues/2021

Possible alternative to https://github.com/alphagov/govuk-design-system/pull/2110/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

This PR:

- Installs [metalsmith/sass](https://github.com/metalsmith/sass) - this requires us to update our version of Metalsmith to the latest version too
- Uninstalls metalsmith-sass and node-sass
- Tweak existing sass config and setup so it continues to work as before
- Sets `quietDeps: true` to suppress deprecation warnings about GOV.UK Frontend sass (which doesn't use Dart Sass yet)

### CODE REVIEW

**Comment by domoscargin** [comment]
🔥 

Can confirm that this resolves the issues on my M1 machine related to https://github.com/alphagov/govuk-design-system/pull/2110/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

These are the steps I took:

1. Deleted node_modules folder
2. Checked this branch out
3. `nvm use` `npm install`
4. Updated `.nvmrc` and `package.json - version` to 16.14.2
5. `nvm use` `npm install`

P'raps @querkmachine or @36degrees could confirm on their machines, but otherwise I'm happy for this PR to wipe the README section mentioned above.

**Comment by querkmachine** [comment]
Can second this. I didn't encounter issues with updating Node following those steps.

**Review by domoscargin — APPROVED** [review]
🥇

File: lib/metalsmith.js [hunk 1]
```diff
@@ -69,7 +69,8 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
 
   // convert *.scss files to *.css
   .use(sass({
-    includePaths: ['node_modules'] // an array of paths that sass can look in when attempt to resolve @import declarations
+    quietDeps: true,
+    loadPaths: ['node_modules', 'src/stylesheets'] // an array of paths that sass can look in when attempt to resolve @import declarations
```
Comment (36degrees) [inline lib/metalsmith.js:73]: Possibly something a bit strange about putting `src/stylesheets` in loadPaths, but can't think of any practical reasons not to…

I guess the alternative would be to make all of the imports of `example-init` relative? I wonder how it ever worked with node-sass 🤔

**Review by 36degrees — APPROVED** [review]
Seems like a sensible change to me 👍🏻 

A slight reservation about src/stylesheets being in loadPaths, but not worth blocking over.

EDIT: Too slow 😅

File: lib/metalsmith.js (hunk 1 above)
Comment (vanitabarrett) [reply lib/metalsmith.js:73]: Woops sorry, think I pressed merge as you reviewed! Yeah I was a bit confused about how this was ever working before. I did try with relative imports but still couldn't seem to get it to work and it felt a bit... messy. 🤔 Maybe worth seeing if this causes any issues before revisiting?

**Pull request merged**